Makefile

```diff
@@ -1,10 +1,12 @@
 BUILD_VERSION := $(shell cat VERSION)
-BUILD_DATE := $(shell date -u '+%Y%m%dT%H%M%SZ')
 
 .PHONY: build
 build:
+	$(eval BUILD_DATE := $(shell date -u '+%Y%m%dT%H%M%SZ'))
+	$(eval BUILD_ARGS := --build-arg BUILD_DATE=${BUILD_DATE} --build-arg BUILD_VERSION=${BUILD_VERSION})
+
 	docker pull digitalmarketplace/base
-	docker build --cache-from digitalmarketplace/base -t digitalmarketplace/base -f base.docker .
+	docker build --pull --cache-from digitalmarketplace/base ${BUILD_ARGS} -t digitalmarketplace/base -f base.docker .
```

Comment:
Showing my docker ignorance - what does the `--pull` option give us here?

Reply:
It makes sure that the latest version of python3.6-slim-stretch is downloaded before building

Reply:
I tried to explain it in the commit message, I'll expand it further so it's clearer

Reply:
I tried to explain it in the commit message, I'll expand it further so it's clearer

Reply:
Doh! So you did - I should read more thoroughly 😸